packages/universal-components/src/Button/GenericButtonWrapper.js

```diff
@@ -26,12 +27,13 @@ export default function Button({
   style,
   testID,
   children,
+  isLoading,
 }: Props) {
   const onPressHandler = onPress || (() => {});
 
   return (
     <Touchable
-      disabled={!onPress || disabled}
+      disabled={!onPress || disabled || isLoading}
```

Comment:
Web button is still enabled during loading. We need to add some similar condition like this native one to `Button.web.js` for both button types ( `<a>` & `<button>` ).

Reply:
I ve changed this condition Button.web.js as well as Button.native.js. And removed one from GenericButtonWrapper as its not added anymore.
But random test errors appear so the check wont pass

Reply:
👍 Thanks

Weird that tests are ok when I run it locally.